examples/progressbar_simpletest.py

```diff
@@ -27,10 +27,10 @@
 
 current_progress = 0.0
 while True:
-    while current_progress <= 1.0:
-        print("Progress: {}%".format(current_progress * 100))
-        progress_bar.progress = current_progress
-        current_progress += 0.05
-        if current_progress >= 1.0:
-            current_progress = 0.0
-        time.sleep(0.01)
+    for current_progress in range(0, 21):
```

Comment:
I don't entirely understand the range ending in 21 as 100% It's more intuitive to me for 100% to be... well... 100 :)

Reply:
Ok, I get what you are doing now that i've read through it a few times. I think the prior example was more intuitive to a newbie (myself) as it made me think in percentages but it does expose the float issue. Maybe leave both in as examples?

Reply:
This change allowed us to count by integers instead of floating point numbers. There was some more discussion about it in the #circuitpython channel on discord this morning if you are interested in learning more about it. http://adafru.it/discord

I think you are right though it would be more clear if it ended on 100. however if it counts by 1 it will be somewhat slower than it is now. we can step by 5 in the range loop though to get the same speed. I'll make this change too. Thank you.

Reply:
Being just the example, I think the slower counting to 100 makes more sense. Maybe include a comment mentioning performance? Thank you for fixing this!